sql/item.h

```diff
@@ -5526,6 +5526,9 @@ class Item_type_holder: public Item,
   TYPELIB *enum_set_typelib;
   Field::geometry_type geometry_type;
 
+  bool sys_trx_start;
+  bool sys_trx_end;
```

Comment:
Use **public** `flags`, don't create accessor members `is_sys_trx_...()`. Access by `flags` member and use `VERS_SYS_START_FLAG`, `VERS_SYS_END_FLAG`.